src/promptflow-evals/promptflow/evals/evaluate/_eval_run.py

```diff
@@ -154,7 +137,31 @@ def _get_scope(self) -> str:
             self._workspace_name,
         )
 
-    def _start_run(self, run_name: Optional[str]) -> bool:
+    def start_run(self) -> None:

```

Comment:
If we don't expect the caller to use this method directly, change it to private.
Even better, we can get rid of start_end and end_run, move the implementation to enter and exit instead.

Reply:
Fixed.